README.md

```diff
@@ -84,6 +216,14 @@ If your form is not backed by a model, use the `bootstrap_form_tag`. Usage of th
 <% end %>
 ```
 
+### Future Compatibility
+
+The Rails team has suggested that `form_for` and `form_tag`
+may be deprecated and then removed in future versions of Rails.
+`bootstrap_form` will continue to support
+`bootstrap_form_for` and `bootstrap_form_tag`
+as long as Rails supports `form_for` and `form_tag`.
```

Comment:
This is a great clarification 👍 

Could you remove the hard line wrap?

Reply:
Sorry. An old `nroff` habit.